contentpacks/generate_dubbed_video_mappings.py

```diff
@@ -35,33 +35,35 @@ def ensure_dir(path):
             raise
 
 
-def download_ka_dubbed_video_csv(download_url=None, cache_filepath=None):
+def download_ka_dubbed_video_csv(download_url=None, is_khan_csv=False, cache_filepath=None):
 
     """
     Function to do the heavy lifting in getting the dubbed videos map.
     Could be moved into utils
     """
     # Get the redirect url
     if not download_url:
-        logging.info("Getting spreadsheet location from Khan Academy")
-        khan_url = "http://www.khanacademy.org/r/translationmapping"
+        csv_url = "http://learningequality.org/r/translationmapping"
+        if is_khan_csv:
+            csv_url = "http://www.khanacademy.org/r/translationmapping"
+        logging.info("Getting spreadsheet location from (%s)" % csv_url)
         try:
-            download_url = urllib.request.urlopen(khan_url).geturl()
+            download_url = urllib.request.urlopen(csv_url).geturl()
             if "docs.google.com" not in download_url:
                 logging.warn("Redirect location no longer in Google docs (%s)" % download_url)
             else:
                 download_url = download_url.replace("/edit", "/export?format=csv")
         except:
             # TODO: have django email admins when we hit this exception
-            raise Exception("Expected redirect response from Khan Academy redirect url.")
+            raise Exception("Expected redirect response from (%s)" % csv_url)
 
     logging.info("Downloading dubbed video data from %s" % download_url)
 
     data = requests.get(download_url)
     attempts = 1
     while data.status_code != 200 and attempts <= 100:
         time.sleep(30)
-        data = requests.get(url)
+        data = requests.get(csv_url)
```

Comment:
Hmm, with the previous code, this loop won't ever succeed because `url` is not declared anywhere.